onnxruntime/python/tools/transformers/models/whisper/README.md

```diff
@@ -55,3 +55,167 @@ $ python3 convert_to_onnx.py -m openai/whisper-tiny --output whispertiny --use_e
 # From wheel:
 $ python3 -m onnxruntime.transformers.models.whisper.convert_to_onnx -m openai/whisper-tiny --output whispertiny --use_external_data_format --precision int8 --quantize_embedding_layer
 ```
+
+## Benchmark Whisper
```

Comment:
Could you also add A100 performance numbers for various versions of the model (tiny, medium, large)?

Reply:
Added